src/test/e2e/bundle_test.go

```diff
@@ -647,6 +651,7 @@ func TestArchCheck(t *testing.T) {
 }
 
 func TestListImages(t *testing.T) {
+	os.Unsetenv("UDS_CONFIG")
```

Comment:
```suggestion
```
good catch cleaning up env vars, since you took care of them in the tests that are setting the envs, i think we can go ahead and delete this one.

Reply:
ah yeah thanks for catching that